client/src/Photo.jsx

```diff
@@ -1,12 +1,99 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
 
-const Photo = ({ photo }) => (
-  <img src={photo.photo_url} alt="home interior" />
-);
+const StyledPhoto = styled.div`
+  grid-column-start: ${(props) => {
```

Comment:
Do you need to deconstruct index for styling?

Reply:
I did try to do it with object destructuring, but it was not working. My understanding is that you do not when using styled-components. I'll add a task to my trello board to look into this more. Thanks!